File: 389. Find the Difference/sol.cpp
class Solution {
public:
    char findTheDifference(string s, string t) {
        int count = 0;
        for( unsigned int i = 0; i < t.length(); i++){
            count += int(t[i]);
        }
        for( unsigned int i = 0; i < s.length(); i++){
            count -= int(s[i]);
        }
        
        return char(count);
    }
};

File: 389. Find the Difference/sol.java
class Solution {
    public char findTheDifference(String s, String t) {
        int count = 0;

        for (int i = 0; i < t.length(); i++) {
            count += (int) t.charAt(i);
        }
        for (int i = 0; i < s.length(); i++) {
            count -= (int) s.charAt(i);
        }

        return (char) count;
    }
}